doc/install.rst

```diff
@@ -28,45 +28,42 @@ You may choose to install ``mne-bids``
 Installation via pip
 --------------------
 
-Besides ``numpy`` and ``scipy`` (which are included in the standard Anaconda
-installation), you will need to install the most recent version of ``MNE``
-using the ``pip`` tool:
+To install MNE-BIDS including all dependencies required to use all features,
+simply run:
 
 .. code-block:: bash
 
-   $ pip install -U mne
+   pip install --user -U mne-bids[full]
 
+This ``pip`` command will also work if you want to upgrade if a newer version
+of ``mne-bids`` is available.
 
-Then install ``mne-bids``\ :
+If you don't require advanced features like interactive visual data inspection,
+you may also install a basic version of MNE-BIDS via
 
 .. code-block:: bash
 
-   $ pip install -U mne-bids
```

Comment:
does the removal of the `$` break the recently added copybutton functionality? see line 45 in conf.py

Reply:
nope, just checked.